apps/pie-docs/src/tokensMetadata.json

```diff
@@ -1054,44 +1054,49 @@
         "category": "fontWeight",
         "displayName": "Regular"
       },
-      "size-a": {
+      "size-12": {
         "category": "fontSize",
-        "displayName": "Font size a",
+        "displayName": "Font size 12",
         "description": "Text size: 12px <br /> Line height: 16px"
       },
-      "size-b": {
+      "size-14": {
         "category": "fontSize",
-        "displayName": "Font size b",
+        "displayName": "Font size 14",
         "description": "Text size: 14px <br /> Line height: 20px"
       },
-      "size-c": {
+      "size-16": {
         "category": "fontSize",
-        "displayName": "Font size c",
+        "displayName": "Font size 16",
         "description": "Text size: 16px <br /> Line height: 24px"
       },
-      "size-d": {
+      "size-19": {
```

Comment:
It seems strange to me that we have a 19px font size. What is this for? (dw this isn't blocking this PR)

Reply:
it's used in the interactive-M alias token. Spoke to Jordi about this awhile back, we're keeping it in for now 👍